docs/standard/io/how-to-write-text-to-a-file.md

```diff
@@ -89,7 +92,8 @@ This topic shows different ways you can write text to a file for .NET Framework
  [!code-vb[OpenFileWindowsStore#Code](../../../samples/snippets/visualbasic/VS_Snippets_CLR/openfilewindowsstore/vb/mainpage.xaml.vb#code)]  
   
 ## See Also  
- <xref:System.IO.StreamWriter>  
+ <xref:System.IO.StreamWriter>
+ <xref:System.IO.Path>
```

Comment:
@isaac2004 please make sure that every line in this section ends with **two** white spaces. Otherwise, lines will appear in one line at the result page output.